gap/callwithtimeout.gd

```diff
@@ -0,0 +1,54 @@
+#############################################################################
+##
+#F  IO_CallWithTimeout( <timeout>, <func>[, <arg1>[, <arg2>....]] )  
+##         . . call a function with a time limit
+#F  IO_CallWithTimeoutList( <timeout>, <func>, <arglist> )  
+##
+##  <#GAPDoc Label="IO_CallWithTimeout">
+##  <Index>Timeouts</Index>
+##  <ManSection>
+##  <Func Name="IO_CallWithTimeout" Arg='timeout, func, .....'/>
+##  <Func Name="IO_CallWithTimeoutList" Arg='timeout, func, arglist'/>
+##
+##  <Description>
+##    <C>IO_CallWithTimeout</C> and <C>IO_CallWithTimeoutList</C> support calling a function
+##  with a limit on length of time it will run. The function is run inside a copy of the current GAP
+##  session, so any changes it makes to global variables are thrown away when the function finishes
+##  or times out.<P/>
+##
+##  <C>IO_CallWithTimeout</C> is variadic. 
+##  Its third and subsequent arguments, if any, are the arguments passed to <A>func</A>.
```

Comment:
How about: "Any arguments to it beyond the first two are passed as argument to <A>func</A>."